### ISSUE TITLE
Support any website...?

### ISSUE THREAD

**Issue by braco** [psychobunny/nodebb-plugin-blog-comments#98]
Wouldn't it be possible to support any website?

The comments would be lazy initialized in the sense that nodebb would only create the post on its back end when a user was entering a first comment on that page.

There is a simple configuration to check if the current page the comment box lives on is valid. Maybe a wildcard for the URL.

"https://mydomain.com/blog/*"

the nodebb plugin would check to see that the URL returns a 200 (not 404) before allowing the first comment.

The use case is that I would like to drop nodebb comments onto a static blog, but don't see why this wouldn't work for everything.

**Comment by psychobunny** [comment]
Yeah! Haha. It was something I realized half way through the build (some
client wanted WP). Yes it could be done that way. Interested in taking on
this build? What’s your app, maybe I might jump in.

Happy holidays, merry Christmas

**Comment by braco** [comment]
Happy holidays!!

I only have a couple of weeks worth of experience in nodebb (as a user), so I'm pretty useless for PRs.

I have a somewhat large nonprofit community that I just launched a website for, and part of that is a ton of scientific content. Each one of the papers has its own page now, so it would be really nice if people could comment under that particular study and have that tracked in nodebb.

Also seems like you could deeply simplify this plugin, doesn't it? This would let it work on any website, WP etc included, no integrations needed.

You could do something as simple as 
```
<meta name="comments" content="enabled">
```
and have the server check for that tag

and then maybe even tags/categorization in the meta as well?
```
<meta name="nodebb-category" content="xyz">
```

**Comment by psychobunny** [comment]
You’re not wrong. This plugin was mostly paid for and designed by those who
wanted xyz use cases, so obviously as I alluded earlier, half way through I
realized that this basically can be refactored properly to essentially
support most platforms.

Now, very interestingly! I may be aligned with your use case. Let’s talk in
the new year? Almost sounds like there’s a particular project I have in
mind that almost sounds like I stole it from you, re: comments on
scientific papers

I’m also from that industry so colour me intrigued. :)

**Comment by braco** [comment]
post holiday bump! 😬

**Comment by psychobunny** [comment]
Happy new years! Shoot me your personal and let’s see what I can do. No
promises but I’d be happy to help and work on this if our interests are
aligned!
